lte/gateway/python/magma/mobilityd/dhcp_client.py

```diff
@@ -57,6 +58,10 @@ def __init__(self,
         self._dhcp_notify = dhcp_wait
         self._dhcp_interface = iface
         self._msg_xid = 0
+        self._lease_renew_wait_min = lease_renew_wait_min
+        self.monitor_thread = threading.Thread(target=self._monitor_dhcp_state)
```

Comment:
should we move this outside of the DHCPClient? I feel like it's better instantiating this from the IPAllocator and then passing it as a dependency on the constructor?

Reply:
some of changes would be covered in other RP refactoring mobilityD, we can revisit it, after that.